apps/main/src/dex/components/PagePool/Swap/index.tsx

```diff
@@ -458,6 +462,7 @@ const Swap = ({
                 loading: userPoolBalancesLoading || isMaxLoading,
                 symbol: fromToken?.symbol,
                 showSlider: false,
```

Comment:
We have showSlider: false everywhere, in the future we should invert the default imo